app/src/main/res/xml/preferences_playback.xml

```diff
@@ -76,6 +76,11 @@
                 android:key="prefPlaybackTimeRespectsSpeed"
                 android:summary="@string/pref_playback_time_respects_speed_sum"
                 android:title="@string/pref_playback_time_respects_speed_title"/>
+        <SwitchPreference
+            android:defaultValue="false"
+            android:key="prefStreamOverDownload"
+            android:summary="@string/pref_stream_over_download_sum"
```

Comment:
The number of spaces does not match the other settings in the file (have a look at td setting above)

Reply:
My bad, not sure how this change got lost (or half-made?) updated again and double-checked the github diff. It should be there now.